packages/vue/src/components/range/MultiRange.jsx

```diff
@@ -76,7 +75,10 @@ const MultiRange = {
 			} else {
 				const currentItems = props.data.filter(value => item.indexOf(value.label) !== -1);
 				currentValue = [...currentValue, ...currentItems];
-				selectedValues = { ...selectedValues, [item]: true };
+				selectedValues = {
+					...selectedValues,
+					[typeof item === 'object' ? item.label : item]: true,
```

Comment:
@mohdashraf010897 What is this change about?

Reply:
when values are selected from the range options they get passed in as objects and not simply 'strings', earlier implementation handles string values but. not objects and that's why I was not able to set values to default for this, here in the current updated implementation both data type objects and strings are accepted.
In short the edge case was unhandled.